llvm/lib/Transforms/Scalar/DFAJumpThreading.cpp

```diff
@@ -820,11 +820,14 @@ struct TransformDFA {
       : SwitchPaths(SwitchPaths), DT(DT), AC(AC), TTI(TTI), ORE(ORE),
         EphValues(EphValues) {}
 
-  void run() {
+  bool run() {
+    bool MadeChange = false;
     if (isLegalAndProfitableToTransform()) {
       createAllExitPaths();
+      MadeChange = true;
       NumTransforms++;
```

Comment:
nit: You could just return true here and return false below. No need for the variable.